packages/devtools-reps/src/object-inspector/index.js

```diff
@@ -175,64 +216,107 @@ class ObjectInspector extends Component {
     return item.path;
   }
 
-  setExpanded(item: Node, expand: boolean) {
```

Comment:
The `expand` method looks like a key to understand the on-demand fetching logic. An explanatory comment (in-front of the method) would help to understand the code.

Reply:
Done